doc/tutorials/space_shooter/step_5.md

```diff
@@ -0,0 +1,80 @@
+# Adding Enemies
+
+Now that the starship is able to shoot, we need something for the player to shoot at! So for
+this step we will work on adding enemies to the game.
+
+So first things first, let's create an `Enemy` class that will represent the enemies in game:
+
+```dart
+class Enemy extends SpriteAnimationComponent
+    with HasGameReference<SpaceShooterGame> {
+  Enemy({
+    super.position,
+  }) : super(size: Vector2.all(50));
+
+  @override
+  Future<void> onLoad() async {
+    await super.onLoad();
+
+    animation = await game.loadSpriteAnimation(
+      'enemy.png',
+      SpriteAnimationData.sequenced(
+        amount: 4,
+        stepTime: .2,
+        textureSize: Vector2.all(16),
+      ),
+    );
+
+    width = 50;
+    height = 50;
+    anchor = Anchor.center;
+  }
+
+  @override
+  void update(double dt) {
+    super.update(dt);
+
+    position.y += dt * 250;
+
+    if (position.y > game.size.y) {
+      removeFromParent();
+    }
+  }
+}
+```
+
+Note how for now, the `Enemy` class is super similar to the `Bullet` one, the only differences are
+their sizes, animation information and bullets travel from bottom to top, while enemies travel from
+top to bottom, so nothing new here.
+
+Next we need to make the enemies spawn on the game, the logic that we will do here will be simple,
+we will simply make enemies spawn from the top of the screen at a random position in the `x` axis.
+
+Once again, we could manually make all the time based event in the game `update` method, maintain
+a random instance to get the enemy x position and so on and so forth, but Flame provides us a
+way to avoid having to write all that by ourselves, we can use the `SpawnComponent`! So in the
+`SpaceShooterGame.onLoad` method let's add the following code:
+
+```dart
+    add(
+      SpawnComponent(
+        factory: (index) {
+          return Enemy();
+        },
+        period: 1,
+        area: Rectangle.fromLTWH(0, 0, size.x, -50),
```

Comment:
Maybe have a constant for the height of the enemy instead of hardcoding -50?

Reply:
added a const in the enemy class since that is where this value was taken from